subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/build/TestProjectInitiation.groovy

```diff
@@ -294,4 +294,11 @@ trait TestProjectInitiation {
         "build.gradle.kts"
     }
 
+    List<TestFile> createDirs(String... dirs) {
+        dirs.collect({ name ->
+            TestFile tf = file(name)
+            tf.mkdir()
```

Comment:
❓ Shouldn't this be `mkdirs()` to handle path with multiple segments in the argument instead?

Reply:
The idea was that it might catch situations where the directory is created relative to the wrong parent... but now that I made all the changes to integration tests it would probably succeed in those cases anyway and instead of this method failing the deprecation warning would still fail the test... so lets change it